FlowCryptCommon/Extensions/StringExtensions.swift

```diff
@@ -61,27 +61,14 @@ public extension NSAttributedString {
 
 // MARK: Email parsing
 public extension String {
-    var userAndRecipientDomain: (user: String, domain: String)? {
+    var email: (username: String?, domain: String?) {
```

Comment:
It would be more ergonomic to return `EmailParts?` where `EmailParts` is `struct { username, domain }`. Then you only have to do one nil check. Also rename `var email` to `var emailParts`?

Reply:
Agree, having one `nil` will be easier to check.